src/sr_state.erl

```diff
@@ -20,8 +20,14 @@
 %%% Types
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+-type options() ::
+  #{ path => string()
+   , model => atom()
```

Comment:
I think `path` and `model` could be defined as required (":="), what do you think @elbrujohalcon ?